datasets/curiosity_dialogs/README.md

```diff
@@ -46,37 +47,172 @@ task_ids:
+  'message': ['Hi. I want information about Namibia.',
+   'Nmbia is a country in southern Africa.',
+   'Do you have information about the media there?',
+   'A mentional amount of foriegn',
+   'What about it?',
+   "Media and journalists in Namibia are represented by the Namibia chapter of the Media Institute of 'southern Africa and the Editors Forum of Namibia.",
+   'Interesting! What can you tell me about the politics and government?',
+   'Namibia formed the Namibian Defence Force, comprising former enemies in a 23-year bush war.',
+   'Do you have more information about it?',
+   "With a small army and a fragile economy , the Namibian government's principal foreign policy concern is developing strengthened ties within the Southern African region.",
+   "That's all I wanted to know. Thank you!",
+   'My pleasure!'],
+  'message_id': ['617343895',
+   '2842515356',
+   '4240816985',
+   '520711081',
+   '1292358002',
+   '3677078227',
+   '1563061125',
+   '1089028270',
+   '1607063839',
+   '113037558',
+   '1197873991',
+   '1399017322'],
+  'sender': [0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 0, 1]},
+ 'related_entities': ['Western Roman Empire',
+  'United Kingdom',
+  'Portuguese language',
+  'Southern African Development Community',
+  'South Africa',
+  'Kalahari Desert',
+  'Namib Desert',
+  'League of Nations',
+  'Afrikaans',
+  'Sub-Saharan Africa',
+  'Portugal',
+  'South-West Africa',
+  'Warmbad, Namibia',
+  'German language',
+  'NBC'],
+ 'reported': 0,
+ 'second_aspect': 'Politics and government',
+ 'shuffle_facts': 1,
+ 'tag': 'round_2',
+ 'user_dialog_rating': 5,
+ 'user_id': 207,
+ 'user_other_agent_rating': 5}
+```
+
+
 
 ### Data Fields
 
-[More Information Needed]
+* `messages`: List of dialogs between the user and the assistant and their associated attributes
+  * `dialog_acts`: List of actions performed in the dialogs
+  * `facts`:  List of facts returned by the assistant
+    * `fid`: Fact ID 
+    * `source`: Source for the fact
+    * `used`:  Whether facts were used before in the same dialog
+  * `liked`: List of values indicating whether each dialog was liked
+  * `message`: List of dialogs (messages) between the user and the assistant
+  * `message_id`: Message ID
+  * `sender`:  Message author ID (numeric)
+* `known_entities`:  Rooted facts about entities the user knows
+* `focus_entity` : Entity in focus in the dialogs
+* `dialog_id `: Dialog ID
+* `inferred_steps`:  Number of inferred steps
+* `created_time`:  Time of creation of the dialog
+* `aspects`:  List of two aspects which the dialog is about
+* `first_aspect`: First aspect
+* `second_aspect`: Second aspect
+* `shuffle_facts`: Whether facts were shuffled
+* `related_entities` : List of fifteen related entities to the focus entity
+* `tag`: Conversation tag
+* `user_id`: User ID
+* `assistant_id`: Assistant ID
+* `is_annotated`:  
+* `user_dialog_rating`:  
+* `user_other_agent_rating`: 
+* `assistant_dialog_rating`: 
+* `assistant_other_agent_rating`: 
```

Comment:
can you add the feature type for the fields with no information ? And maybe the value range ? (probably 1-5 for the ratings, and 0 or 1 for `is_annotated `)

Reply:
Sure. Do I need to put something like `More Information Needed` or `Incomplete Information` to mention that the descriptions are incomplete?

Reply:
Yes feel free to also mention explicitly that information is missing for those fields